grid-proxy/pkg/types/nodes.go

```diff
@@ -141,7 +141,8 @@ type NodeFilter struct {
 	OwnedBy           *uint64  `schema:"owned_by,omitempty"`
 	Rented            *bool    `schema:"rented,omitempty"`
 	RentedBy          *uint64  `schema:"rented_by,omitempty"`
-	AvailableFor      *uint64  `schema:"available_for,omitempty"`
+	RentableBy        *uint64  `schema:"rentable_by,omitempty"`   // rented by twin or rentable
```

Comment:
RentedBy instead of RentableBy no?

Reply:
`rentedby` is already a functional filter returns all the nodes rented by a specific twin.
`rentableby` is a new filter that should return all the nodes rented by a specific twin + all the nodes available to rent

Reply:
`rentedby` is already a functional filter returns all the nodes rented by a specific twin.
`rentableby` is a new filter that should return all the nodes rented by a specific twin + all the nodes available to rent

Reply:
`available_for` returns the nodes rented by a twin id + the shared nodes. (available to deploy on)
`rentable_for` returns the nodes rented by a twin id + the rentable nodes. (rented or available to rent)

Reply:
Tb bos :D let's make it rentable_or_rented_by :D